### PR TITLE
Task/3703 new hosting dashboard

### PR DESCRIPTION
**Pull request by tiwariav**

### CODE REVIEW

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -172,6 +172,27 @@ msgstr "CHF/Monat"
 msgid "Start VM"
 msgstr "VM jetzt starten"
 
+msgid "My Dashboard"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Mein Dashboard"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -172,6 +172,27 @@ msgstr "CHF/Monat"
...
+
+msgid "Create VM"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "VM erstellen"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -172,6 +172,27 @@ msgstr "CHF/Monat"
...
+
+msgid "My VMs"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Meine VMs"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -172,6 +172,27 @@ msgstr "CHF/Monat"
...
+
+msgid "My SSH Keys"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Meine SSH Keys"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -172,6 +172,27 @@ msgstr "CHF/Monat"
...
+
+msgid "My Bills"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Meine Rechnungen"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -172,6 +172,27 @@ msgstr "CHF/Monat"
...
+
+msgid "My Settings"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Meine Einstellungen"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -172,6 +172,27 @@ msgstr "CHF/Monat"
...
+
+msgid "Support / Contact"
+msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Support / Kontakt"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -482,6 +503,19 @@ msgid ""
 "contact Data Center Light Support."
 msgstr ""
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Kontaktiere den Data Center Light Support"

File: hosting/static/hosting/img/24-hours-support.svg [hunk 9]
```diff
@@ -1,3 +1,20 @@
+<<<<<<< HEAD
```
Comment (pcoder) [inline hosting/static/hosting/img/24-hours-support.svg]: Something seems not good here. <<<<<<< HEAD - did it come from a diff tool?

File: hosting/static/hosting/img/billing.svg [hunk 10]
```diff
@@ -1 +1,17 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" class="symbol symbol-billing" aria-labelledby="title" role="img"><title id="title">billing icon</title><g data-name="Layer 1"><path class="cls-1" d="M.37.023v15.954l2.775-1.387 2.775 1.387L8 14.59l2.775 1.387 2.081-1.387 2.775 1.387V.023zm13.873 13.709l-1.487-.744-2.081 1.387L7.9 12.989l-2.08 1.387-2.675-1.337-1.387.694V1.41h12.485z" role="presentation"/><path class="cls-1" d="M4.206 3.617h7.741v1.348H4.206zm0 2.697h7.741v1.349H4.206zm0 2.697h7.741v1.349H4.206z" role="presentation"/></g></svg>
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (pcoder) [inline hosting/static/hosting/img/billing.svg]: Same here

File: hosting/static/hosting/img/24-hours-support.svg (hunk 9 above)
Comment (tiwariav) [reply hosting/static/hosting/img/24-hours-support.svg]: it seems there was a merge conflict for the new images, went unnoticed. Thanks for pointing out.

**Comment by verysanghee** [comment]
attention

**Pull request merged**